packages/manager/apps/web-office/src/api/users/key.ts

```diff
@@ -0,0 +1,3 @@
+export const getOfficeUsersQueryKey = (serviceName: string) => [
+  `get/license/office/${serviceName}/user`,
```

Comment:
**nitpick**: for query keys prefer splitting the string like `['get', 'license', 'office', ...]` it will allow you to invalidate multiple query for example if you invalidate `['get']` it will invalidate all queries starting with `['get']` and so on

Reply:
It's fixed